boards: arm: frdm_kw41z: fix PWM LEDs period

gmarull opened the pull request:
The period was set to 0, a value not meaningful to drive an LED. A value
of 20 msec has been chosen as most other boards do.

Prep work for https://github.com/zephyrproject-rtos/zephyr/pull/44523

gmarull committed:
boards: arm: frdm_kw41z: fix PWM LEDs period

The period was set to 0, a value not meaningful to drive an LED. A value
of 20 msec has been chosen as most other boards do.

Signed-off-by: Gerard Marull-Paretas <gerard.marull@nordicsemi.no>

anangl commented on boards/arm/frdm_kw41z/frdm_kw41z.dts:
```diff
@@ -49,15 +49,15 @@
-			pwms = <&tpm2 0 0 0>;
+			pwms = <&tpm2 0 PWM_MSEC(20) 0>;
 			label = "User PWM LD1";
 		};
 		green_pwm_led: pwm_led_1 {
-			pwms = <&tpm2 1 0 0>;
+			pwms = <&tpm2 1 PWM_MSEC(20) 0>;
 			label = "User PWM LD2";
 		};
 		red_pwm_led: pwm_led_2 {
-			pwms = <&tpm0 2 0 0>;
+			pwms = <&tpm0 2 PWM_MSEC(20) 0>;
```
Shouldn't also the `flags` cells be corrected to `PWM_POLARITY_INVERTED`? All these LEDs are active low according to the `leds` node above.

gmarull replied:
right, good catch! fixed

gmarull committed:
dts: arm: nxp: nxp_kw4{0,1}z: include PWM dt-bindings

In order to be consistent with other platforms, include the PWM
dt-bindings by default.

Signed-off-by: Gerard Marull-Paretas <gerard.marull@nordicsemi.no>

gmarull committed:
boards: arm: frdm_kw41z: fix PWM LEDs polarity

The PWM driven LEDs are active low, so the polarity has to be inverted.

Signed-off-by: Gerard Marull-Paretas <gerard.marull@nordicsemi.no>
